src/amo/components/AddonMoreInfo/index.js

```diff
@@ -101,7 +105,9 @@ export class AddonMoreInfoBase extends React.Component<InternalProps> {
       statsLink = (
         <Link
           className="AddonMoreInfo-stats-link"
-          href={`/addon/${addon.slug}/statistics/`}
+          href={addQueryParams(`/addon/${addon.slug}/statistics/`, {
+            src: location.query.src,
```

Comment:
what would this resolve to when there isn't a `src`?  (same question in other places)

Reply:
This will be `undefined` because `location.query` is a sort of `dict` (JS object) and the `src` key won't exist, so that will be `undefined`.